### ISSUE TITLE
MaterialSkin icon

### ISSUE THREAD

**Issue by CDrummond** [codechimp-org/lms-tvh#21]
MaterialSkin currently has a SVG icon for this plugin, and a JSON file that is used to map from its PNG icon to the SVG icon. As of MaterialSkin 2.3.0, plugin's can ship this SVG icon themselves.

To support this:

1. Rename `plugins/TVH/icon.png` to `plugins/TVH/icon_svg.png`
2. Copy https://raw.githubusercontent.com/CDrummond/lms-material/master/MaterialSkin/HTML/material/html/images/tvheadend.svg to `plugins/TVH/icon.svg`

...and it should all work. What happens is that if Material finds an icon named xxx_svg.png it assumes there is a matching xxx.svg in the same folder.

If you do make this change, let me know and I'll remove the entry from Material's JSON mapping file.

--------------------

For more info on this icon mapping see: https://github.com/CDrummond/lms-material/wiki/Plugin-icon-mapping

**Comment by andrew-codechimp** [comment]
Thanks for the info and a neat solution.
I have made these changes and just released 1.0.95 with this in.

As far as I can tell your mapping is cutting in before the icon name checking so I can't totally validate that it works.

I've also changed the icon name in install.xml to match the new icon_svg.png which may be worth putting as a reminder in your wiki page for those that don't maintain plugins often.
